### PR TITLE
Additions to explain unnecessary Nav (UI) SDK Gradle declarations

### PR DESCRIPTION
**Pull request by langsmith**
Per @Guardiola31337 's suggestion, this pr adds some more clarifying text about unnecessary Gradle declarations; that the Nav UI SDK _already_ includes the Maps and Nav SDKs.
Screen Shot 2019-06-26 at 1 55 55 PM
Screen Shot 2019-06-26 at 2 01 05 PM

### CODE REVIEW

File: src/pages/navigation/overview/index.md [hunk 1]
```diff
@@ -103,6 +103,11 @@ Learn how to install the core Navigation SDK and request your first route using
 
 {{<Note title="Add the core Navigation SDK only when not using the Navigation UI SDK" imageComponent={<BookImage size="60" />}>}}
 If you're using the Navigation UI SDK, you **don't** have to declare the Mapbox Navigation SDK as well. If you only declare the Navigation UI SDK in your project's Gradle file, the Mapbox Navigation SDK will automatically be included.
+
+{{</Note>}}
+
+{{<Note title="Maps SDK included" imageComponent={<BookImage size="60" />}>}}
```
Comment (Guardiola31337) [inline src/pages/navigation/overview/index.md]: 👀 Maps SDK is only included in Mapbox Navigation UI SDK

Comment (langsmith) [reply src/pages/navigation/overview/index.md]: Fixed

File: src/pages/navigation/overview/index.md [hunk 2]
```diff
@@ -235,7 +236,14 @@ The Navigation UI SDK is the fastest way to get a navigation UI into your applic
...
+This dependency is different from the one used to compile the core Mapbox Navigation SDK, but it will still include everything from the core Mapbox Navigation SDK. 
+
+{{<Note title="Maps and Navigation SDKs included" imageComponent={<BookImage size="60" />}>}}
+
+<!--copyeditor ignore best-->
+If you're using the Navigation UI SDK, we recommend you **don't** include an explicit version of the Maps SDK or Navigation SDK in your build.gradle file. The Mapbox Navigation UI SDK already includes the core Mapbox Navigation SDK and Mapbox Maps SDK for Android. Gradle automatically takes care of adding the right dependencies. The Navigation UI SDK will include the Navigation and Maps SDK versions that have been tested to provide the best user and developer experience for navigation experiences. Manually adding a different versions can lead to unexpected behavior.
```
Comment (colleenmcginnis) [inline src/pages/navigation/overview/index.md]: >Manually adding a different versions can lead to unexpected behavior.

to 

>Manually adding different versions can lead to unexpected behavior.

?

Comment (langsmith) [reply src/pages/navigation/overview/index.md]: fixed

**Pull request merged**